Project.toml

```diff
@@ -14,17 +14,19 @@ Thermodynamics = "b60c26fb-14c3-4610-9d3e-2d17fe7ff00c"
 [weakdeps]
 Cloudy = "9e3b23bb-e7cc-4b94-886c-65de2234ba87"
 DataFrames = "a93c6f00-e57d-5684-b7b6-d8193f3e46c0"
+EnsembleKalmanProcesses = "aa8a2aa5-91d8-4396-bcef-d4f2ec43552d"
 MLJ = "add582a8-e3ab-11e8-2d5e-e98b27df1bc7"
 
 [extensions]
 CloudyExt = "Cloudy"
-EmulatorModelsExt = ["DataFrames", "MLJ"]
+EmulatorModelsExt = ["DataFrames", "MLJ", "EnsembleKalmanProcesses"]
 
 [compat]
 ClimaParams = "0.10"
 Cloudy = "0.4"
 DataFrames = "1.6"
 DocStringExtensions = "0.8, 0.9"
+EnsembleKalmanProcesses = "1.1.5"
```

Comment:
Would it be enough to set it to "1.1"?

Reply:
Yes, thanks!

Reply:
Sounds good. Do we need Optim? Or is it a dependence of EKP or something else?